site/content/en/docs/tasks/run/rayservices.md

```
@@ -77,6 +92,6 @@ The RayService looks like the following:
 {{< include "examples/jobs/ray-service-sample.yaml" "yaml" >}}
 
 {{% alert title="Note" color="primary" %}}
-The example above comes from [here](https://raw.githubusercontent.com/ray-project/kuberay/v1.4.2/ray-operator/config/samples/ray-service.sample.yaml)
-and only has the `queue-name` label added and requests updated.
+The example above comes from [here](https://raw.githubusercontent.com/ray-project/kuberay/v1.7.0/ray-operator/config/samples/ray-service.sample.yaml)
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)